.github/workflows/pr-arm.yml

```
@@ -0,0 +1,102 @@
+name: "PR Tests (aarch64)"
+permissions: read-all
+
+# Trigger for PR and merge to develop branch
+on:
+  push:
+    branches: develop
+  pull_request:
+  workflow_dispatch:
+
+env:
+  CTEST_OUTPUT_ON_FAILURE: 1
+  LAPACK_VERSION: 3.12.0
+  ARMPL_VERSION: 25.04.1
+  DPCPP_VERSION: nightly-2025-04-01
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-24.04-arm
+    # One runner for each domain
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - config: oneMath BLAS
```

```suggestion
        - config: ArmPL BLAS
```
I think we should also rename `oneMath BLAS` in `pr.yml` back to `oneMKL BLAS`.